server/src/main/java/org/solarbank/server/MonetaryAmountSerializer.java

```diff
@@ -0,0 +1,19 @@
+package org.solarbank.server;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import javax.money.MonetaryAmount;
+
+public class MonetaryAmountSerializer extends JsonSerializer<MonetaryAmount> {
```

Comment:
As with CurrencyUnitSerializer, added this so only the amount is returned in the response instead of whole MonetaryAmount object